I think the Travis CI failure is related:
```
go test -mod=vendor -ldflags "-X github.com/cilium/cilium/pkg/kvstore.consulDummyConfigFile=/tmp/cilium-consul-certs/cilium-consul.yaml -X github.com/cilium/cilium/pkg/testutils.CiliumRootDir=/home/travis/gopath/src/github.com/cilium/cilium -X github.com/cilium/cilium/pkg/datapath.DatapathSHA=1234567890abcdef7890 -X github.com/cilium/cilium/pkg/logging.DefaultLogLevelStr="error"" github.com/cilium/cilium/bugtool -test.v -timeout 360s -coverprofile=coverage.out -covermode=count -coverpkg ./... || exit 1; tail -n +2 coverage.out >> coverage-all-tmp.out;
/bin/bash: 2: unbound variable
Makefile:152: recipe for target 'unit-tests' failed
make: *** [unit-tests] Error 1
The command "./.travis/build.sh" exited with 2.
```
It was also failing with the same error on the previous build, for aarch64 only as well.

--- a/Makefile
+++ b/Makefile
@@ -146,9 +146,14 @@ ifeq ($(SKIP_VET),"false")
 endif
 	$(QUIET) echo "mode: count" > coverage-all-tmp.out
 	$(QUIET) echo "mode: count" > coverage.out
-	$(QUIET)$(foreach pkg,$(patsubst %,github.com/cilium/cilium/%,$(TESTPKGS)),\
-		$(_GO) test $(GOFLAGS) $(TEST_UNITTEST_LDFLAGS) $(pkg) $(GOTEST_BASE) $(GOTEST_COVER_OPTS) || exit 1; \
-		tail -n +2 coverage.out >> coverage-all-tmp.out;)
+	# It seems that in some env if the path is large enough for the full list
+	# of files, the full bash command in that target gets too big for bash and
+	# hence will trigger an error of too many arguments. As a workaround, we
+	# have to process these packages in different subshells.
+	for pkg in $(patsubst %,github.com/cilium/cilium/%,$(TESTPKGS)); do \
+		$(_GO) test $(GOFLAGS) $(TEST_UNITTEST_LDFLAGS) $$pkg $(GOTEST_BASE) $(GOTEST_COVER_OPTS) || exit 1; \
+		tail -n +2 coverage.out >> coverage-all-tmp.out; \
+	done
 	$(MAKE) generate-cov
 	@rmdir ./daemon/1 ./daemon/1_backup 2> /dev/null || true
 	$(MAKE) stop-kvstores

